farbenmeer/ausguck
# feat(docs): autofill readme on commit

Hard overengineered but I had fun doing it :)

BTW: Should be tested on Mac as there are sometimes small differences in CLI commands(flags, arguments, etc.)

How to test:

1. Check out the repository
2. `npm install`
3. `touch ./decks/$SOMETALK.mdx` (replace `$SOMETALK` with a valid filename)
4. `git add .`
5. `git commit -m 'testing'
6. look at the `README.md` and see if your talk is listed there.
7. Don't push ;)

resolves #5
